dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -101,7 +101,9 @@ RUN python3 -m pip install aiohttp \
                  scapy==2.4.5 \
                  setuptools-rust \
                  six \
-                 snappi[ixnetwork,convergence]==0.11.16 \
+                 snappi==0.9.1 \
```

Comment:
In the dockerfile, we install python pkgs for both python2/python3/python3-venv
If you change the package. Please keep all of them the same and synced.